sklearn/multioutput.py

```diff
@@ -404,8 +404,22 @@ def fit(self, X, Y, sample_weight=None, **fit_params):
         self.classes_ = [estimator.classes_ for estimator in self.estimators_]
         return self
 
-    @property
-    def predict_proba(self):
+    def _check_predict_proba(self):
+        if hasattr(self, "estimators_") and not all(
+            [hasattr(estimator, "predict_proba") for estimator in self.estimators_]
```

Comment:
```suggestion
        if not hasattr(self, "estimators_") or hasattr(self, "estimators_") and not all(
            [hasattr(estimator, "predict_proba") for estimator in self.estimators_]
```

Reply:
@glemaitre Do we need to add `not hasattr(self, "estimators_")` here?

Reply:
I think that the previous way is easier to get to be honest.

Reply:
but we want to raise an error when `estimators_` attribute doesn't exist, right?